src/components/create-courses/component/authors-lists.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {useMemo} from "react";
+import { Button, List, Col, Row } from "reactstrap";
+
+export default function AuthorsLists({ authorsList, setAuthorsList }) {
+
+	const authorsNotSelected = useMemo( () => {
+		return authorsList.filter(({ selected }) => !selected);
+	}, [authorsList])
+
+	const authorsSelected = useMemo( () => {
+		return authorsList.filter(({ selected }) => selected);
+	}, [authorsList])
+
+	function selectToggle(authorId) {
+		const newArray = authorsList.filter((author) => author.id !== authorId);
+		const author = authorsList.find(author => author.id === authorId);
+		author.selected === true ?	author.selected = false : author.selected = true;
+		newArray.push(author)
+		setAuthorsList(newArray)
+	}
+
+	return <Row xs='1' className='mt-1'>
+		{!authorsNotSelected.length ? <List type="unstyled">
```

Comment:
Use `isEmpty` from lodash.isEmpty and optional chaining in case of operating data to prevent page breaking

Reply:
This memo list Was removed